### PR TITLE
Internal: Kit-Library - Show react query dev tools only in elementor debug mode [ED-3505]

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

File: core/app/modules/kit-library/assets/js/app.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const queryClient = new QueryClient( {
 	},
 } );
 
-const isDebug = elementorCommon.config.isDebug || false;
+const isElementorDebug = elementorCommon.config.isElementorDebug || false;
```
Comment (ItzhakAvm) [inline core/app/modules/kit-library/assets/js/app.js:20]:
```suggestion

```

Check out below...

File: core/app/modules/kit-library/assets/js/app.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ export default function App() {
 						<Overview path="/overview/:id"/>
 					</Router>
 				</SettingsProvider>
-				{ isDebug && <ReactQueryDevtools initialIsOpen={ false }/> }
+				{ isElementorDebug && <ReactQueryDevtools initialIsOpen={ false }/> }
```
Comment (ItzhakAvm) [inline core/app/modules/kit-library/assets/js/app.js:34]:
```suggestion
				{ elementorCommon.config.isElementorDebug && <ReactQueryDevtools initialIsOpen={ false }/> }
```

File: core/app/modules/kit-library/assets/js/app.js (hunk 1 above)
Comment (Nevoss) [reply core/app/modules/kit-library/assets/js/app.js]: Done

File: core/app/modules/kit-library/assets/js/app.js (hunk 2 above)
Comment (Nevoss) [reply core/app/modules/kit-library/assets/js/app.js]: Done

**Review by YitzhakAvm — APPROVED** [review]

**Review by matipojo — APPROVED** [review]

**Pull request merged**